docs/worker-node/install-singularity.md

```diff
@@ -222,17 +232,34 @@ Set these options in `/etc/singularity/singularity.conf`:
     Look for `singularity.conf.rpmnew` after upgrades and merge in any
     changes to the defaults.
 
+#### Configuring the RPM to be unprivileged ####
+
+If you choose to run the RPM unprivileged, after
+[enabling unprivileged singularity](#enabling-unprivileged-singularity),
+change the line in `/etc/singularity/singularity.conf` that says
+`allow setuid = yes` to
+
+        allow setuid = no
+
+Note that the setuid-root executables stay installed, but they will exit
```

Comment:
"will exit very early" -- does that mean they'll error out? Or do you just mean that they'll drop privileges early? Should users chmod the executables after setting this option?

Reply:
Yes, they will error out.  There's a non-setuid copy that runs when 'allow setuid = no'.   The point of this statement is that it's save to not do a chmod.

Reply:
It would be less confusing if you say that non-setuid equivalents of the executables exist and will be run instead. The current paragraph makes it sound like singularity would break because nothing would run.

Reply:
Ok I will add a sentence.